package.json

```diff
@@ -70,7 +70,7 @@
     "extends": "./node_modules/@s-ui/lint/stylelint.config.js"
   },
   "dependencies": {
-    "@iabtcf/core": "1.0.1",
+    "@iabtcf/core": "1",
```

Comment:
To be discussed with the team. Are we sure to enable the automatic update to any 1 version? It's not preferable to enable de dependandbot to make the autobump and check if any test was broken?

Reply:
For me it's OK to use dependantbot too

Reply:
@paulusrex added dependabot in this PR and fixed the iab's dependency